libc-test/build.rs

```diff
@@ -333,6 +356,12 @@ fn main() {
             "ptrace" |
             "sigaltstack" if rumprun => true,
 
+            // uses size_t instead of socklen_t
+            "getnameinfo" if dragonfly => true,
```

Comment:
I think it is already addressed by #110. OpenBSD had same problem and I splitted `getnameinfo` implementations with `size_t` and `socklen_t`